test/lodash.ts

```diff
@@ -0,0 +1,13 @@
+import {assert} from "chai";
+
+describe("lodash", () => {
```

Comment:
Same here, declare do `declare var _: any;` before this line to prevent IDE errors.